### DISCUSSION TITLE
Onclick removed from scripts

### DISCUSSION THREAD

**Discussion by sapiennervosa** [requarks/wiki#5031] [Error / Bug Report]
I've been trying to develop embedded tools in my wiki pages using JS but it seems that whenever I create a button with an 'onclick' property it gets stripped out when the page is rendered.

For example, this line in the editor:

`<button onclick="alert('Hello world!')">Test OnClick</button>`

Is rendered in the browser as this:
`<button>Test OnClick</button></div></template> `

Is the cause of this some sort of sanitizing that occurs during rendering? If so, it greatly diminishes the usefulness of scripts. Is it possible to disable this behavior?

**Comment by JuL314** [comment]
Hi !
I'm a bit late but I found this : 

https://github.com/requarks/wiki/discussions/4122#discussioncomment-917512

I had the same issue, took me a few hours to figure it out haha
